website/static/js/fangorn.js

```diff
@@ -2533,16 +2543,26 @@ function _dropLogic(event, items, folder) {
     tb.syncFileMoveCache[folder.data.provider] = tb.syncFileMoveCache[folder.data.provider] || {};
     tb.moveStates = [];
 
+
+    // pendingReadyFiles is incremented/decremented after a ready request's response is received, syncFileMoveCache is
+    // popped when a request is sent.
+    tb.pendingReadyFiles = toMove.ready.length;
+
+
     if (toMove.ready.length > 0) {
         tb.syncFileMoveCache[folder.data.provider].ready = tb.syncFileMoveCache[folder.data.provider].ready || [];
         if (SYNC_UPLOAD_ADDONS.indexOf(folder.data.provider) !== -1) {
             toMove.ready.forEach(function(item) {
                 tb.syncFileMoveCache[folder.data.provider].ready.push({'item' : item, 'folder' : folder});
             });
         } else {
-            toMove.ready.forEach(function(item) {
-                doItemOp.call(tb, copyMode === 'move' ? OPERATIONS.MOVE : OPERATIONS.COPY, folder, item, undefined, 'replace');
-            });
+            MOVE_INTERVAL = setInterval(function() {
+                if(toMove.ready.length > 0) {
+                    doItemOp.call(tb, copyMode === 'move' ? OPERATIONS.MOVE : OPERATIONS.COPY, folder, toMove.ready.pop(), undefined, undefined);
+                } else {
+                    MOVE_INTERVAL = clearInterval(MOVE_INTERVAL);
+                }
+            }, MILLSECOND_PER_MOVE_REQUEST);
```

Comment:
This is happening because [this line](https://github.com/CenterForOpenScience/osf.io/pull/8961/files#diff-cc758c0b6e0e49cbf0b40be21b5e6002L2558) was removed. So conflicts were only being called after "readys" were completed [here](https://github.com/CenterForOpenScience/osf.io/pull/8961/files#diff-cc758c0b6e0e49cbf0b40be21b5e6002R739). So because their were no readys, conflicts were never displayed.